api/Navigator.json

```diff
@@ -1713,8 +1714,7 @@
               "version_added": true
             },
             "ie": {
-              "version_added": true,
-              "notes": "Always returns <code>undefined</code>."
+              "version_added": false
```

Comment:
Confirmed with http://mdn-bcd-collector.appspot.com/tests/api/Navigator/productSub, which amounts to `'productSub' in navigator`.

If there had been a getter there, but it always returned undefined, that would have been worth a note, since 'productSub' in navigator` would then have been true, which would mess with feature detection.